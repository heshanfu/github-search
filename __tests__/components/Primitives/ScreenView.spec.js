import renderer from 'react-test-renderer';
import 'jest-styled-components';
import React from 'react';

import ScreenView from '../../../app/components/Primitives/ScreenView';

describe('<ScreenView />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ScreenView />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
