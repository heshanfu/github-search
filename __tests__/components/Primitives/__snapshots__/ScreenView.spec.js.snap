// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScreenView /> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "paddingBottom": 20,
        "paddingLeft": 20,
        "paddingRight": 20,
        "paddingTop": 20,
      },
      undefined,
    ]
  }
/>
`;
